### ISSUE TITLE
Not working on OSX High Sierra 

### ISSUE THREAD

**Issue by benjivo** [iseekwonderful/csgoGlow#281]
Error when putting executable in Terminal 

Found CSGO PID: 5820
Found the Client.dylib address: 0x128d06000 
Module should end at 0x12a22978c

any help is appreciated, thanks.

**Comment by ethano8225** [comment]
Did you use my version of the cheat? Did you make sure to use sudo? If you’re on Mojave, did you make Wall/terminal able to control the computer? (Mine doesn’t work on battle royale game mode, you have to use a different version, I’ll link it to you if that’s the problem).

**Comment by flxwu** [comment]
I am on Mojave and for me it gives the same output, but it doesn't work for me too

**Comment by benjivo** [comment]
Yes to all, and only used it in casual, still nothing. Also I’m on high Sierra as listed in the title.

**Comment by ethano8225** [comment]
try to inject after you join a game if you didn’t 
I haven’t used it in awhile so it may just be broken temporarily
I’ll try it tmrw

**Comment by benjivo** [comment]
Works after the new update by ning!

**Comment by THUGREX** [comment]
I don't have enough space to download xcode so can anyone try to send the Wall.exec file to me via email?? Thanks you so much :D thuginthan@gmail.com

**Comment by ramanamatya** [comment]
> Works after the new update by ning!

where is the new update ?
